dotnet/src/Connectors/Connectors.AzureAIInference/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Should there be a .proj file in this PR and also changes to the main solution?

Reply:
This was merged already in a previous PR.